### DISCUSSION TITLE
Erreur 500 après mise à jour

### DISCUSSION THREAD

**Discussion by adrienloyat** [dbarzin/mercator#992] [Q&A] [answered]
Bonjour
J'ai voulu mettre à jour à la dernière version suite à ma question d'hier. J'ai un peu galéré étant donné qu'il a fallu faire une montée de version de php de 8.1 à 8.2. 
Tout semble fonctionner sauf la page de paramètre qui renvoie une erreur 500. Je n'ai aucune trace dans les logs apache.
Est-ce que vous avez une idée ?

**Comment by dbarzin** [comment, marked as answer]
La page était manquante, tu peux faire un `git pull`.
Pour la prochaine fois, les logs se trouvent dans `/var/www/mercator/storage/logs/laravel.log`
